### DISCUSSION TITLE
JS Script

### DISCUSSION THREAD

**Discussion by desplogt** [EvgeniyMukhamedjanov/liquid-ajax-cart#66] [Q&A]
I created a script that makes changes within the cart, every time it updates the elements they refresh but the JS script is not executed again, is there a function to be able to carry out this action?

**Comment by EvgeniyMukhamedjanov** [comment]
@desplogt 
You can use the [subscribeToCartSectionsUpdate](https://liquid-ajax-cart.js.org/reference/subscribeToCartSectionsUpdate/) to register your function. Your function will be called every time when the cart is updated.
